decidim-debates/app/commands/decidim/debates/admin/create_debate.rb

```diff
@@ -36,6 +39,18 @@ def create_debate
             feature: form.current_feature
           )
         end
+
+        def send_notification_to_space_followers
+          Decidim::EventsManager.publish(
+            event: "decidim.events.debates.debate_created",
+            event_class: Decidim::Debates::CreateDebateEvent,
+            resource: debate,
+            recipient_ids: form.current_feature.participatory_space.followers.pluck(:id),
```

Comment:
Shouldn't you use `debate.participatory_space` too here?

Reply:
It's exactly the same 🤔